src/test/java/no/unit/nva/model/RoleDtoTest.java

```diff
@@ -69,4 +79,43 @@ public void isValidReturnsFalseWhenUsernameIsNullOrBlank(String emptyOrNullRoleN
         setter.invoke(roleDto, emptyOrNullRoleName);
         assertThat(roleDto.isValid(), is(equalTo(false)));
     }
+
+    @DisplayName("RoleDto object contains type with value \"Role\"")
```

Comment:
Hmm. We need to discuss testing strategy

Reply:
You can try converting it back to POJO instead.